src/lwc/geBatchWizard/geBatchWizard.html

```diff
@@ -67,10 +68,9 @@ <h2 class="slds-modal__title">
                                                        size="12">
                                     <c-util-expandable-section id={section.id}
                                                                label={section.label}>
-
                                         <template iterator:it={section.elements}>
 
-                                            <div key={it.value.apiName}
+                                            <div key={it.value.id}
```

Comment:
This is intentional and fixes a console warning that appears. apiName is undefined always